db-scripts/src/main/resources/migration.sql

```diff
@@ -426,3 +426,16 @@ FROM `gene`);
 
 UPDATE info SET DB_SCHEMA_VERSION="2.4.1";
 -- ========================= end of reference genes related tables ========================================================================
+
+##version: 2.5.0
+
+CREATE TABLE `fraction_genome_altered` (
+  `CANCER_STUDY_ID` int(11) NOT NULL,
+  `SAMPLE_ID` int(11) NOT NULL,
+  `VALUE` double NOT NULL,
+  KEY (`CANCER_STUDY_ID`,`SAMPLE_ID`),
+  FOREIGN KEY (`CANCER_STUDY_ID`) REFERENCES `cancer_study` (`CANCER_STUDY_ID`) ON DELETE CASCADE,
+  FOREIGN KEY (`SAMPLE_ID`) REFERENCES `sample` (`INTERNAL_ID`) ON DELETE CASCADE
+);
+
```

Comment:
We will need migration code to fill the data for existing samples.

Reply:
I'll see if I can convert it to a SQL-only calculation (it's most likely), if not, we might need to calculate the values for every study and then export insert statements of values (a long list of inserts).